api-admin/src/main/java/bio/terra/pearl/api/admin/service/kit/KitExtService.java

```diff
@@ -81,41 +55,74 @@ public void addException(Exception exception) {
   public KitRequestDto requestKit(
-      AdminUser adminUser,
-      String studyShortcode,
-      String enrolleeShortcode,
+      PortalEnrolleeAuthContext authContext,
       KitRequestService.KitRequestCreationDto kitRequestCreationDto) {
-    Enrollee enrollee = authUtilService.authAdminUserToEnrollee(adminUser, enrolleeShortcode);
-    return kitRequestService.requestKit(adminUser, studyShortcode, enrollee, kitRequestCreationDto);
+    return kitRequestService.requestKit(
+        authContext.getOperator(),
+        authContext.getStudyShortcode(),
+        authContext.getEnrollee(),
+        kitRequestCreationDto);
+  }
+
+  @EnforcePortalStudyEnvPermission(permission = "BASE")
+  public KitRequestListResponse requestKits(
+      PortalStudyEnvAuthContext authContext,
+      List<String> enrolleeShortcodes,
+      KitRequestService.KitRequestCreationDto kitRequestCreationDto) {
+    KitRequestListResponse response = new KitRequestListResponse();
+    for (String enrolleeShortcode : enrolleeShortcodes) {
+      try {
+        KitRequestDto kitDto =
+            requestKit(
+                PortalEnrolleeAuthContext.of(
+                    authContext.getOperator(),
+                    authContext.getPortalShortcode(),
+                    authContext.getStudyShortcode(),
+                    authContext.getEnvironmentName(),
+                    enrolleeShortcode),
+                kitRequestCreationDto);
+        response.addKitRequest(kitDto);
+      } catch (Exception e) {
+        // add the enrollee shortcode to the message for disambiguation.  Once we refine the UX for
+        // this, a structured response might be useful here
+        response.addException(new Exception(enrolleeShortcode + ": " + e.getMessage(), e));
+      }
+    }
+    return response;
+  }
+
+  @EnforcePortalEnrolleePermission(permission = "BASE")
+  public KitRequest collectKit(
+      PortalEnrolleeAuthContext authContext, KitRequestService.KitCollectionDto kitCollectionDto) {
+
+    KitRequest kitRequest =
+        kitRequestService.findByEnrolleeAndBarcode(
+            authContext.getEnrollee(), kitCollectionDto.kitBarcode());
+
+    return kitRequestService.collectKit(
+        authContext.getOperator(), kitRequest, KitRequestStatus.COLLECTED_BY_STAFF);
   }
 
-  public Collection<KitRequestDto> getKitRequests(AdminUser adminUser, String enrolleeShortcode) {
-    Enrollee enrollee = authUtilService.authAdminUserToEnrollee(adminUser, enrolleeShortcode);
-    return kitRequestService.findByEnrollee(enrollee);
+  @EnforcePortalEnrolleePermission(permission = "BASE")
+  public Collection<KitRequestDto> getKitRequests(PortalEnrolleeAuthContext authContext) {
+    return kitRequestService.findByEnrollee(authContext.getEnrollee());
   }
 
-  public void refreshKitStatuses(
-      AdminUser adminUser,
-      String portalShortcode,
-      String studyShortcode,
-      EnvironmentName environmentName)
+  @EnforcePortalStudyEnvPermission(permission = "BASE")
```

Comment:
I don't think for this PR, but should we create a new perm for this? `kit_request_view` and `kit_request_edit`?